homeassistant/components/climate/__init__.py

```diff
@@ -51,6 +51,21 @@
 STATE_HEAT_PUMP = 'heat_pump'
 STATE_GAS = 'gas'
 
+SUPPORT_CURRENT_TEMPERATURE = 1 << 0
```

Comment:
Supported features are only used for services/commands the device exposes, not for values that it reports.

Reply:
Oh, didn't realize that. However, at least the eq3btsmart platform cannot display the current temperature (currently it returns the target temperature as current temperature), there may be more.  I thought it would be nice to hide the (wrong) current temperature in the front end in those cases. Do you suggest that we somehow else indicate that the device does not provide a current temperature?

Reply:
On the backend side, it's simply a matter of not implementing `current_temperature`, and let the default implementation return `None`. It's certainly possible however the frontend needs some fixes for that case.

https://github.com/home-assistant/home-assistant/blob/6ad62a2ccbf42fc3c48a6d81917cc91572bf143f/homeassistant/components/climate/__init__.py#L549-L552

Reply:
Okay, then I'll remove `SUPPORT_CURRENT_TEMPERATURE` as well as `SUPPORT_CURRENT_HUMIDITY`, and make the front end hide the values if they are `None`.